scripts/sync_templates.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# Copies job templates from community-templates repository to
+# app/views/templates/ so they can be seeded on new installations.
+#
+# Part of the release process.
+
+REPO=$(mktemp -d)
+trap "rm -rf $REPO" EXIT
+
+git clone -q -b develop https://github.com/theforeman/community-templates $REPO/ct
+
+# Move into destination dir if run from plugin root
+[ -d app/views/templates ] && cd app/views/templates
+
+# Add underscore prefix to snippets
+[ -d $REPO/ct/jobs/snippets ] && (cd $REPO/ct/jobs/snippets && prename -f 's/^([^_])/_$1/' *)
```

Comment:
Oh, sorry, it's some utility bundled with perl on Debian only.  I forget how I got it on my Fedora box, it wasn't packaged.  It's not a bug deal, I'll rewrite it something more portable.